modules/navier_stokes/test/tests/auxkernels/peclet-number-functor-aux/fv-thermal.i

```diff
@@ -0,0 +1,196 @@
+mu = 1
+rho = 1
+k = 1
+cp = 1
+
+[GlobalParams]
+  velocity_interp_method = 'rc'
+  advected_interp_method = 'average'
```

Comment:
maybe add a note that bcs of the Peclet number, this scheme is OK ?

Reply:
```suggestion
  # Maximum cell Peclet number is ~.1 so energy transport is stable without upwinding
  advected_interp_method = 'average'
```